.github/ISSUE_TEMPLATE/simeng-bug-or-issue-report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: SimEng Bug or Issue Report
+about: Report a bug or un-expected SimEng behaviour
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Check List**
+ [ ] The binary I am trying to run has been compiled statically for either RV64 or AArch64.
```

Comment:
Could ask them to run `file` on the binary and provide the output (this is more to force them to do it and prevent them from submitting an issue to hastily). Should answer the first 2 questions here

Reply:
I think the check list and running `file` have merits, will keep the checks and add a new question for `file` below